languages/common-lisp/exercises/concept/basic-lists/.meta/design.md

```diff
@@ -0,0 +1,72 @@
+This issue describes how to implement the `basic-lists` concept exercise for the Common Lisp track.
```

Comment:
The name `basic-lists` implies are sort of hierarchy. As per [the docs](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts), we don't use a hierarchy for concepts. What if you'd just name this `lists`? More advanced list exercises could then be named after the more advanced list concepts they discuss.

Reply:
concept & exercise both renamed - thanks.